frontend/src/components/VLicense/VLicenseElements.vue

```diff
@@ -58,10 +61,16 @@ export default defineComponent({
     const uiStore = useUiStore()
     const isMobile = computed(() => !uiStore.isDesktopLayout)
 
+    const getLicenseDescription = (element: string) => {
```

Comment:
The license keys in en.json5 were converted to camelCase, so we need to use `camel` method for the element (by-sa -> bySa)